imagetest/test_suites/storage_perf/storage_perf_utils.go

```diff
@@ -0,0 +1,128 @@
+package storageperf
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+const (
+	vmName = "vm"
+	// iopsErrorMargin allows for a small difference between iops found in the test and the iops value listed in public documentation.
+	iopsErrorMargin = 0.97
+	// hyperdiskSize is used to determine which partition is the mounted hyperdisk.
+	hyperdiskSize = 100
+	bootdiskSize  = 10
+	mountDiskName = "hyperdisk"
+	// TODO: Set up constants for compute.Disk.ProvisionedIOPS int64, and compute.Disk.ProvisionedThrougput int64, then set these fields in appendCreateDisksStep
+)
+
+type fioOutput struct {
+	jobs []fioJob               `json:"jobs,omitempty"`
+	X    map[string]interface{} `json:"-"`
+}
+
+type fioJob struct {
+	readResult  fioStatistics          `json:"read,omitempty"`
+	writeResult fioStatistics          `json:"write,omitempty"`
+	X           map[string]interface{} `json:"-"`
+}
+
+type fioStatistics struct {
+	iops      float64                `json:iops,omitempty"`
+	bandwidth float64                `json:bw_mean,omitempty"`
+	X         map[string]interface{} `json:"-"`
+}
+
+type blockDeviceList struct {
+	blockDevices []blockDevice `json:"blockdevices,omitempty"`
+}
+
+type blockDevice struct {
+	name       string `json:"name,omitempty"`
+	size       string `json:"size,omitempty"`
+	deviceType string `json:"type,omitempty"`
+	// Other fields are not currently used.
+	X map[string]interface{} `json:"-"`
+}
+
+func getMountDiskPartition(diskExpectedSizeGb int) (string, error) {
+	lsblkCmd := "lsblk"
```

Comment:
Even if we do it in future iterations, it's worth stubbing out where we need to split Linux and Windows logic. This would be one place. And we could either do this with the function (this one is Linux specific) or inside the function.

Reply:
currently commented so that windows logic can be added in a future iteration